src/vs/workbench/parts/terminal/electron-browser/terminal.contribution.ts

```diff
@@ -0,0 +1,143 @@
+			},
+			'integratedTerminal.ansiColors.cyan': {
+				'description': nls.localize('integratedTerminal.ansiColors.cyan', "Cyan color for terminals that support ANSI escape sequences (format: #rrggbb)"),
+				'type': 'string',
+				'default': '#0598bc'
+			},
+			'integratedTerminal.ansiColors.white': {
+				'description': nls.localize('integratedTerminal.ansiColors.white', "White color for terminals that support ANSI escape sequences (format: #rrggbb)"),
+				'type': 'string',
+				'default': '#e5e5e5'
+			},
+			'integratedTerminal.ansiColors.brightBlack': {
+				'description': nls.localize('integratedTerminal.ansiColors.brightBlack', "Bright black color for terminals that support ANSI escape sequences (format: #rrggbb)"),
+				'type': 'string',
+				'default': '#000000'
+			},
+			'integratedTerminal.ansiColors.brightRed': {
+				'description': nls.localize('integratedTerminal.ansiColors.brightRed', "Bright red color for terminals that support ANSI escape sequences (format: #rrggbb)"),
+				'type': 'string',
+				'default': '#cd3131'
+			},
+			'integratedTerminal.ansiColors.brightGreen': {
+				'description': nls.localize('integratedTerminal.ansiColors.brightGreen', "Bright green color for terminals that support ANSI escape sequences (format: #rrggbb)"),
+				'type': 'string',
+				'default': '#09885a'
+			},
+			'integratedTerminal.ansiColors.brightYellow': {
+				'description': nls.localize('integratedTerminal.ansiColors.brightYellow', "Bright yellow color for terminals that support ANSI escape sequences (format: #rrggbb)"),
+				'type': 'string',
+				'default': '#e5e510'
+			},
+			'integratedTerminal.ansiColors.brightBlue': {
+				'description': nls.localize('integratedTerminal.ansiColors.brightBlue', "Bright blue color for terminals that support ANSI escape sequences (format: #rrggbb)"),
+				'type': 'string',
+				'default': '#0451a5'
+			},
+			'integratedTerminal.ansiColors.brightMagenta': {
+				'description': nls.localize('integratedTerminal.ansiColors.brightMagenta', "Bright magenta color for terminals that support ANSI escape sequences (format: #rrggbb)"),
+				'type': 'string',
+				'default': '#bc05bc'
+			},
+			'integratedTerminal.ansiColors.brightCyan': {
+				'description': nls.localize('integratedTerminal.ansiColors.brightCyan', "Bright cyan color for terminals that support ANSI escape sequences (format: #rrggbb)"),
+				'type': 'string',
+				'default': '#0598bc'
+			},
+			'integratedTerminal.ansiColors.brightWhite': {
+				'description': nls.localize('integratedTerminal.ansiColors.brightWhite', "Bright white color for terminals that support ANSI escape sequences (format: #rrggbb)"),
+				'type': 'string',
+				'default': '#e5e5e5'
+			}
+		}
+	});
+
+	// Register Service
+	registerSingleton(ITerminalService, TerminalService);
+
+	// Register Output Panel
+	(<panel.PanelRegistry>Registry.as(panel.Extensions.Panels)).registerPanel(new panel.PanelDescriptor(
+		'vs/workbench/parts/terminal/electron-browser/terminalPanel',
```

Comment:
@Tyriar if you have an AMD entry point like this, it needs to be declared in our buildfile.js (see https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/buildfile.js#L34)